xla/tests/build_defs.bzl

```diff
@@ -111,7 +111,7 @@ def xla_test(
         if backend == "cpu":
             backend_deps = ["//xla/service:cpu_plugin"]
             backend_deps += ["//xla/tests:test_macros_cpu"]  # buildifier: disable=list-append
-        elif backend == "gpu":
+        elif backend == "gpu" or backend == "gpu_a100" or backend == "gpu_v100" or backend == "gpu_p100" or backend == "gpu_h100":
```

Comment:
I think it'd be nice to have a `gpu_backends` list at the top with `all_backends`, and then redefine like `all_backends = gpu_backends + ["cpu"] + plugins.keys()`

Reply:
Incorporated review comments.